golang/sig0/doh.go

```diff
@@ -64,6 +64,7 @@ func FindDOHEndpoint(name string) (*url.URL, error) {
 		return nil, fmt.Errorf("findDOHEndpoint: no answer section for %s", lookup)
 	}
 
+	// TODO deal with more than one SVCB "_dns." + name in RRSet
```

Comment:
good call!

we should start making a list of things we need static test vectors for to drop our need for network i/o during tests and this would definitely be one of them.